compiler/one-cmds/onelib/OptionBuilder.py

```diff
@@ -60,7 +60,7 @@ def _build_optimize(self, commands):
                 options.extend(['--' + k])
                 continue
             for opt in CONSTANT.OPTIMIZATION_OPTS:
-                if k == opt and v == "True":
+                if k == opt[0] and v == "True":
```

Comment:
(this is not related with this PR but... @mhs4670go , iv `v` is not related with this for loop, can we hoist `v == 'True'` outside of this loop? in another PR)

Reply:
Sorry. I don't get it. Could you explain what you said with simple example?

Reply:
as `v` is not related any variable inside the loop,
```py
if v == "True":
  for opt in CONSTANT.OPTIMIZATION_OPTS:
    if k == opt[0]:
      options.extend(['--' + k])
      break
```

Reply:
We can:) I will post a PR soon.